saas/static/js/djaodjin-saas-vue.js

```diff
@@ -1954,6 +1954,16 @@ new Vue({
         url: djaodjinSettings.urls.user.api_accessibles,
         typeaheadUrl: djaodjinSettings.urls.api_candidates,
     },
+    methods: {
```

Comment:
The code to accept a request for a subscription look like this (same file):
```
        acceptRequest: function(organization, request_key) {
            var vm = this;
            var url = (djaodjinSettings.urls.organization.api_profile_base +
                organization + "/subscribers/accept/" + request_key + "/");
            vm.reqPut(url, function (){
                vm.get();
            });
        },
```
In both cases `acceptGrant` or `acceptRequest`, the code should be identical except for the API end point being called.